File: app/src/main/java/pwr/barwa/chat/data/dto/ChannelDto.kt
package pwr.barwa.chat.data.dto

data class ChannelDto(
    val id: Long,
    val name: String,
    val members: List<Long>,
    val messages: List<Long>,
    val lastMessage: MessageDto?,
    val isGroup: Boolean,
    val image: String? = null,
    val created: Long
)

File: Chat-WebServer/AM-Chat-WebServer/Data/DTOs/ChannelDTO.cs
namespace AM_Chat_WebServer.Data.DTOs;

public class ChannelDTO
{
    public long Id { get; set; }
    public List<long> Members { get; set; } = new List<long>();
    public List<long> Messages { get; set; } = new List<long>();
    public string Name { get; set; } = string.Empty;
}